client/src/components/SystemStatus/SystemStatus.module.scss

```diff
@@ -0,0 +1,42 @@
+.content {
+  /* WARNING: Mimicked on: History, Allocation, DataFiles, DataFilesProjectsList, DataFilesProjectFileListing, PublicData */
+  padding-top: 1.75rem; /* ~28px (22.5px * design * 1.2 design-to-app ratio) */
+  padding-left: 1.5em; /* ~24px (20px * design * 1.2 design-to-app ratio) */
+
+  /* As a flex child */
+  flex-grow: 1;
+  height: 85%;
+  width: 100% !important;
+}
+
+.sysmon-content {
+  /* WARNING: Mimicked on: History, Allocation, DataFiles, DataFilesProjectsList, DataFilesProjectFileListing, PublicData */
+  padding-top: 1.75rem; /* ~28px (22.5px * design * 1.2 design-to-app ratio) */
+  padding-left: 1.5em; /* ~24px (20px * design * 1.2 design-to-app ratio) */
+
+  /* As a flex child */
+  flex-grow: 1;
+  height: 15%;
+  width: 100% !important;
+}
+
+.section {
+  width: 50% !important;
```

Comment:
**`!important` Alert**

<details><summary>Are you sure <code>!important</code> is necessary?</summary>

It is like [The Death Star of CSS](https://confluence.tacc.utexas.edu/download/attachments/101384277/css-specificity-wars.png?version=1&modificationDate=1579973712320&api=v2). Please read about [how **to** use and **not** use `!important`](https://confluence.tacc.utexas.edu/display/UP/UI+-+CSS+Style+Guide+-+Specificity#UICSSStyleGuideSpecificity-!important). If `!important` is required, add a comment that explains what code you are fighting with it.

</details>

Reply:
This CSS is replaced in #816.